tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_provision.c

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 #include "mesh_test.h"
+#include <mesh/net.h>
```

Comment:
Should be `"mesh/net.h"`, as only the include folder is treated as a system header directory